### ISSUE TITLE
[Request] Permissions Per Potion

### ISSUE THREAD

**Issue by DaringShepard** [DieReicheErethons/Brewery#218]
Would it be possible to get specific permission nodes for each potion? Currently, there is no way to limit specific potions to certain user groups/classes, and it would be phenomenal if that could be changed. 

Example usage:
>    1:
>       name: Ruined Haste Potion/Haste Potion/Strong Haste Potion
>       ingredients:
>       - Sugar/5
>       - Feather/1
>       - Nether_Wart/1
>       cookingtime: 3
>       color: ORANGE
>       difficulty: 3
>       alcohol: 0
>       effects:
>       - FAST_DIGGING/1-3/20-60/
>       **permission: brewery.potion.HastePotion**

The permission node, when added to the config of a potion, would then be able to be used in the permissions plugin of your choice to allow/deny a group to brew said potion.

**Comment by Sn0wStorm** [comment]
This is probably not going to work as there is no keeping track of who put which ingredients into the cauldron and keeping track of who was involved in the process of brewing

**Comment by Valaron-Jakob** [comment]
There is a bit of a workaround for that with the upcoming update. Use an item as ingredient which only a player with a specific permission can craft.

**Comment by Sn0wStorm** [comment]
> There is a bit of a workaround for that with the upcoming update. Use an item as ingredient which only a player with a specific permission can craft.

Yes that is how i thought how you could control who can brew what, as it is closer to Minecraft vanilla, don't let people geht ahold of an item needed for brewing and they can't brew it.
Otherwise i could add permission nodes for adding certain ingredients into cauldron?

**Comment by DaringShepard** [comment]
> Otherwise I could add permission nodes for adding certain ingredients into cauldron

I'd rather not restrict items to certain classes/permission groups, so if able you were able to do this, that'd be fantastic!

**Comment by GeneralSarcasam** [comment]
This would be a very interesting feature indeed, I would love to have something like this on my server!

**Comment by SuperEms** [comment]
Similar vein, but it would also be useful to be able to restrict recipe permissions per world. It would be a great feature to have on my server, so I could restrict brewing permissions per world.
